lib/acts_as_paranoid/core.rb

```diff
@@ -36,7 +36,12 @@ def delete_all!(conditions = nil)
       end
 
       def delete_all(conditions = nil)
-        where(conditions).update_all(["#{paranoid_configuration[:column]} = ?", delete_now_value])
+        ar = where(conditions)
+        if paranoid_configuration[:touch]
+          ar.update_all(["#{paranoid_configuration[:column]} = ?, updated_at = ?", delete_now_value, Time.now])
```

Comment:
What do you think about using the `delete_now_value` instead of `Time.now`? That way it doesn't assume that the paranoid config column has a `Time` value (it could also be a string or boolean).